system/jlib/jthread.cpp

```diff
@@ -1322,7 +1322,7 @@ static void CheckAllowedProgram(const char *prog,const char *allowed)
         if (WildMatch(head.str(),list.item(i)))
             return;
     }
-    ERRLOG("Unauthorized pipe program(%s)",head.str());
+    OERRLOG("Unauthorized pipe program(%s)",head.str());
```

Comment:
AERRLOG would be better.

Reply:
@ghalliday do you mean IERRLOG?

Reply:
@shamser no I mean (a new function) AERRLOG  - which should be reported to the audit audience.
This is something that should be added to an audit trail since it could be someone attempting to bypass security.